docs/apache-airflow-providers-databricks/connections/databricks.rst

```diff
@@ -57,10 +58,19 @@ Password (optional)
 
 Extra (optional)
     Specify the extra parameter (as json dictionary) that can be used in the Databricks connection.
-    This parameter is necessary if using the *PAT* authentication method (recommended):
+
+    Following parameter is necessary if using the *PAT* authentication method (recommended):
 
     * ``token``: Specify PAT to use.
 
+    Following parameters are necessary if using authentication with AAD token:
+
+    * ``azure_client_id``: ID of the Azure Service Principal
```

Comment:
Why don't we put the client_id and client_secret into the user and password fields of the connection? Ultimately, it's just a user and a password.